compass/landice/tests/ensemble_generator/plot_ensemble.py

```diff
@@ -0,0 +1,217 @@
+        yearsOutput = DS['daysSinceStart'].values[:] / 365.0
+        indices = np.nonzero(yearsOutput >= targetYear)[0]
+        if len(indices) > 0:
+            ii = indices[0]
+
+            thickness = DS['thickness'].values
+            bedTopo = DS['bedTopography'].values
+            xCell = DS['xCell'].values[0,:]
+            yCell = DS['yCell'].values[0,:]
+
+            triang = tri.Triangulation(xCell, yCell)
+
+
+            axMaps.tricontour(triang, thickness[0], [1.0], colors='k')
+            axMaps.tricontour(triang, thickness[ii], [1.0], colors='r')
+
+            ii5 = np.nonzero(yearsOutput >= 10.0)[0][0]
+
+            grd0 = ((thickness[0,:]*910.0/1028.0+bedTopo[0,:])>0.0)*(thickness[0,:]>0.0)
+            grd5 = ((thickness[ii5,:]*910.0/1028.0+bedTopo[ii5,:])>0.0)*(thickness[ii5,:]>0.0)
+            grd = ((thickness[ii,:]*910.0/1028.0+bedTopo[ii,:])>0.0)*(thickness[ii,:]>0.0)
+            axMaps.tricontour(triang, grd0, [0.5], colors='k')
+            axMaps.tricontour(triang, grd5, [0.5], colors='c')
+            axMaps.tricontour(triang, grd, [0.5], colors='b')
+
+
+
+
+        f.close()
+
+# plot results
+
+markerSize = 100
+
+fig = plt.figure(1, figsize=(14, 11), facecolor='w')
+nrow=2
+ncol=2
+
+axSLR = fig.add_subplot(nrow, ncol, 1)
+plt.title(f'SLR at year {targetYear} (mm)')
+plt.xlabel('von Mises stress threshold (kPa)')
+plt.ylabel('basal friction law exponent')
+plt.grid()
+plt.scatter(vmThresh, fricExp, s=markerSize, c=SLRAll, plotnonfinite=False)
+if labelRuns:
+    for i in range(nRuns):
+        plt.annotate(f'{runs[i][3:]}', (vmThresh[i], fricExp[i]))
+badIdx = np.nonzero(np.isnan(SLRAll))[0]
+plt.plot(vmThresh[badIdx], fricExp[badIdx], 'kx')
+plt.colorbar()
+
+axArea = fig.add_subplot(nrow, ncol, 2)
+plt.title(f'Total area change at year {targetYear} (km$^2$)')
+plt.xlabel('von Mises stress threshold (kPa)')
+plt.ylabel('basal friction law exponent')
+plt.grid()
+plt.scatter(vmThresh, fricExp, s=markerSize, c=areaChangeAll, plotnonfinite=False)
+badIdx = np.nonzero(np.isnan(areaChangeAll))[0]
+plt.plot(vmThresh[badIdx], fricExp[badIdx], 'kx')
+plt.colorbar()
+
+axGrdArea = fig.add_subplot(nrow, ncol, 3)
+plt.title(f'Grounded area change at year {targetYear} (km$^2$)')
+plt.xlabel('von Mises stress threshold (kPa)')
+plt.ylabel('basal friction law exponent')
+plt.grid()
+plt.scatter(vmThresh, fricExp, s=markerSize, c=grdAreaChangeAll, plotnonfinite=False)
+badIdx = np.nonzero(np.isnan(grdAreaChangeAll))[0]
+plt.plot(vmThresh[badIdx], fricExp[badIdx], 'kx')
+plt.colorbar()
+
+#axfltArea = fig.add_subplot(nrow, ncol, 4)
+#plt.title(f'Floating area change at year {targetYear} (km$^2$)')
+#plt.xlabel('von Mises stress threshold (kPa)')
+#plt.ylabel('basal friction law exponent')
+#plt.grid()
+#plt.scatter(vmThresh, fricExp, s=markerSize, c=fltAreaChangeAll, plotnonfinite=False)
+#badIdx = np.nonzero(np.isnan(fltAreaChangeAll))[0]
+#plt.plot(vmThresh[badIdx], fricExp[badIdx], 'kx')
+#plt.colorbar()
```

Comment:
I will leave this up to you all but I prefer not to have commented-out code unless there is a high likelihood that it will be added back soon.  It would be better to have a config option to plot this, and have it set to `False` by default.

Reply:
I've deleted it.